packages/vite/src/node/__tests__/utils.spec.ts

```diff
@@ -44,6 +45,49 @@ describe('bareImportRE', () => {
   })
 })
 
+describe('deepImportRE', () => {
+  test('should match regular package with subpath', () => {
+    expect(deepImportRE.test('my-lib/schemas')).toBe(true)
+    const match = deepImportRE.exec('my-lib/schemas')
+    expect(match?.[1]).toBe('my-lib')
+    expect(match?.[2]).toBeUndefined()
```

Comment:
```suggestion
    const match = deepImportRE.exec('my-lib/schemas')
    expect(match?.[1]).toBe('my-lib')
    expect(match?.[2]).toBeUndefined()
```
`.test` is covered by `.exec`.